src/server/examples/examples_test.go

```diff
@@ -341,3 +342,107 @@ func TestFruitStand(t *testing.T) {
 	require.NoError(t, c.DeleteRepo("data", false))
 }
+
+func TestScraper(t *testing.T) {
+	if testing.Short() {
+		t.Skip("Skipping integration tests in short mode")
+	}
+	c := getPachClient(t)
+
+	// Check that scraper pipeline manifest appears in the docs (before we create a new repo or anything)
+	readme, err := ioutil.ReadFile("../../../doc/examples/scraper/README.md")
+	require.NoError(t, err)
+	inputPipelineManifest := `
+{
+  "pipeline": {
+    "name": "scraper”
+  },
+  "transform": {
+    "cmd": [ "wget",
+        "--recursive",
+        "--level", "1",
+        "--accept", "jpg,jpeg,png,gif,bmp",
+        "--page-requisites",
+        "--adjust-extension",
+        "--span-hosts",
+        "--no-check-certificate",
+        "--timestamping",
+        "--directory-prefix",
+        "/pfs/out",
+        "--input-file", "/pfs/urls/urls"
+    ],
+    "acceptReturnCode": [4,5,6,7,8]
+  },
+  "parallelism": "1",
+  "inputs": [
+    {
+      "repo": {
+        "name": "urls"
+      }
+    }
+  ]
+}
+`
+	require.Equal(t, true, strings.Contains(string(readme), inputPipelineManifest))
+
+	// create "urls" repo and check that it exists
+	cmd := exec.Command("pachctl", "create-repo", "urls")
+	require.NoError(t, cmd.Run())
+	var pfsRepos []*pfsclient.RepoInfo
+	pfsRepos, err = c.ListRepo([]string{})
+	require.NoError(t, err)
+	require.EqualOneOf(t, toRepoNames(pfsRepos), "urls")
+
+	// Create first commit in repo
+	var output []byte
+	output, err = exec.Command("pachctl", "start-commit", "urls", "master").Output()
+	require.NoError(t, err)
+	require.Equal(t, "master/0\n", string(output))
+	// Check that it's there
+	var commitInfos []*pfsclient.CommitInfo
+	commitInfos, err = c.ListCommit(
+		// fromCommits (only use commits to the 'urls' repo)
+		[]*pfsclient.Commit{{
+			Repo: &pfsclient.Repo{"urls"},
+		}},
+		nil, // provenance (any provenance)
+		client.CommitTypeWrite, // Commit hasn't finished, so we want "write"
+		pfsclient.CommitStatus_NORMAL,
+		true)
+	require.NoError(t, err)
+	require.Equal(t, 1, len(commitInfos))
+	require.Equal(t, "urls", commitInfos[0].Commit.Repo.Name)
+	require.Equal(t, "master/0", commitInfos[0].Commit.ID)
+
+	// Example input takes ~10 minutes to scrape, so just use two URLs here
+	cmd = exec.Command("pachctl", "put-file", "urls", "master/0", "urls")
+	cmd.Stdin = bytes.NewBuffer([]byte("www.google.com\nwww.example.com\n"))
+	require.NoError(t, cmd.Run())
+
+	// finish commit
+	require.NoError(t, exec.Command("pachctl", "finish-commit", "urls", "master/0").Run())
+	b := &bytes.Buffer{}
+	require.NoError(t, c.GetFile("urls", "master/0", "urls", 0, 0, "", false, &pfsclient.Shard{}, b))
+
+	// Create pipeline
+	cmd = exec.Command("pachctl", "create-pipeline", "-f", "-")
+	cmd.Stdin, err = os.Open("../../../doc/examples/scraper/scraper.json")
+	require.NoError(t, err)
+	require.NoError(t, cmd.Run())
+	c.FlushCommit([]*pfsclient.Commit{commitInfos[0].Commit}, nil) // Wait until the URLs have been scraped
```

Comment:
Even if you're not using the returned commitInfos, may as well make sure `FlushCommit` didn't err